CRM/Core/Payment/BaseIPN.php

```diff
@@ -483,23 +483,6 @@ public function completeTransaction(&$input, &$ids, &$objects, $transaction = NU
     CRM_Contribute_BAO_Contribution::completeOrder($input, $ids, $objects, $transaction);
   }
 
-  /**
-   * Get site billing ID.
-   *
-   * @param array $ids
-   *
-   * @return bool
-   */
-  public function getBillingID(&$ids) {
```

Comment:
I think we should deprecate rather than remove this

Reply:
@eileenmcnaughton That's fine. Please let me know about the other comment I made and I will update this